query/search_intersections.js

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable */
```

Comment:
What doesn't eslint like in this file? Ideally we remove this line and fix the thing it complains about.